articles/android-studio-shimmer-loading-effect/index.md

```diff
@@ -185,11 +191,11 @@ Here is the code to implement that.
 
```

Comment:
No need for the extra <androidx.constraintlayout.widget.ConstraintLayout/> in the above layout